lib/lcd.lua

```diff
@@ -133,7 +137,11 @@ end
 -- Erases the screen. --
 function LCD:clear()
     self:command(codes.LCD_CLEARDISPLAY)
+<<<<<<< HEAD
+    cord:await(storm.os.invokeLater, 50*storm.os.MILLISECOND)
+=======
     cord:await(storm.os.invokeLater, 2*storm.os.MILLISECOND)
+>>>>>>> d2f7de8428d1a0ef6d729e0671f624a0a17f7e8a
 end
```

Comment:
This is a merge conflict. You should _never_ commit code with an unresolved merge conflict